sanic/log.py

```diff
@@ -2,12 +2,23 @@
 import sys
 
 from enum import Enum
-from typing import Any, Dict
+from typing import TYPE_CHECKING, Any, Dict
 from warnings import warn
 
 from sanic.compat import is_atty
 
 
+# Python 3.11 changed the way Enum formatting works for mixed-in types.
+if sys.version_info < (3, 11, 0):
+
+    class StrEnum(str, Enum):
+        pass
+
+else:
+    if not TYPE_CHECKING:
```

Comment:
Why do we care about `TYPE_CHECKING` here?

Reply:
Without this, `mypy` errors that `StrEnum` is previously declared.

Reply:
@ahopkins what's the status of this?

Reply:
If you are on 3.11, you need `StrEnum` whether type checking or not. Am I missing something?

Reply:
`mypy` does not seem to care about the previous `if` checking for the version and will complain about `StrEnum` being declared twice.